packages/ng-excel-builder/src/excel-builder.service.ts

```diff
@@ -155,27 +160,40 @@ export class ExcelBuilderService {
                     }
 
                     private getWidthOfColumn(
-                        key: string,
-                        entries: PlainObject[],
+                        title: string,
+                        entries: string[],
                         parameters: Nullable<ColumnParameters>
                     ): number {
                         const { minColumnWidth }: StyleSizes = this.sizes;
 
                         if (parameters?.width === ColumnWidth.MAX_WIDTH) {
-                            return this.calcMaxWidthByEntries(entries, key);
+                            return this.calcMaxWidthByEntries(entries, title);
                         } else if (typeof parameters?.width === 'number') {
                             return parameters.width;
                         } else {
                             return minColumnWidth;
                         }
                     }
 
-                    private calcMaxWidthByEntries(entries: PlainObject[], key: string): number {
-                        const maxLength: number = entries.reduce((length: number, entry: PlainObject): number => {
-                            const currentLength: number = (entry[key]?.toString() ?? '').length;
+                    private calcMaxWidthByEntries(entries: string[], title: string): number {
+                        const titleLength: number = this.getWidthOfString(title, 'bold');
+                        // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+                        const padding: number = this.sizes.fontWidth * 2;
```

Comment:
create variable for `2`, what it is? add comments please why

Reply:
I couldn't think of a name for it for a long time yesterday) but now I guessed it is indent measured In symbols